app/config.local.js

```diff
@@ -29,3 +34,4 @@ kubernetes: "dev-mode"
 };
 
 window.MOCK_ORIGIN_SERVICES = false;
+window.MOCK_SERVICES_DATA = true;
```

Comment:
So the idea of the mock services is to provide a service that mimics the services in origin-web-common. One flag for all to be in mock mode.
The mock data service should be the same as the DataService in origin-web-common and register itself as "DataService" in the 'mockServices' module. When the window.MOCK_ORIGIN_SERVICES flag is true, then the app uses the 'mockServices' module rather than the 'openshiftCommon' module and all the injection of services will be mock services.

Is the mockDataService ready to be used as a mock for the DataService in origin-web-common?